docs/administer/api_rbac.md

```diff
@@ -0,0 +1,40 @@
+# Navigating the API changes for RBAC
+
+Starting with Percona Everest v1.2.0, breaking changes are being made to the API for monitoring instances and backup storage resources. These changes are:
+
+- Before the release of Percona Everest 1.2.0, these resources were globally scoped, but now they will be specific to namespaces. 
+
+- The database clusters will only be able to use monitoring instances and backup storages that are located within the same namespace as the cluster. The system used a `.spec.allowedNamespaces` field to control access to these global resources. This field determined the namespaces where the resource could be accessed, providing a certain degree of access control.
+
+- With the update to Percona Everest v1.2.0, the shift from global scoping to namespacing these resources marks a significant change in how access control is managed, enhancing security by ensuring that resources are only accessible within their designated namespaces.
+
+## Challenges of globally spaced namespaces
+
+The use of the `allowedNamespaces` field for globally scoped resources presented challenges when integrating with the core RBAC model. In order to resolve this inconsistency and align with the RBAC framework, backup storage and monitoring instances are now namespaced resources. This change ensures that all resources adhere to the same RBAC model, resulting in a more consistent and manageable access control structure across the entire system.
```

Comment:
I think we should start by mentioning that 1.2.0 introduces RBAC, and then describe how the current model poses challenges when integrating with it

Reply:
I feel that this is a heavily trimmed down version of the ["Why?"](https://www.notion.so/percona/Breaking-API-changes-in-Everest-v1-2-0-619c0a3540bb4038bb84e2d493b61646?pvs=4#303826c1e4084f7798f636e4d7263bc7) section from the Notion doc. It would be great if you'd consider taking a look again and see if we can paste it here as-is (or with any changes to the wording you feel is necessary..).

IMO this information is important because as a developer, I wouldn't be expecting a breaking API change for a minor upgrade, so it's important we provide adequate justification 😅

Reply:
Fixed